src/types/BooleanSBVR.ts

```diff
@@ -0,0 +1,31 @@
+import * as _ from 'lodash'
+
+const typeFunc = (necessity:string, index:string, defaultValue:string = ' DEFAULT 0') => {
```

Comment:
Can you add a space after each `:` for types please, ie `necessity: string, index: string, ...`